drivers/otp/Kconfig.emu

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2025 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+config OTP_EMULATOR
+	bool "OTP memory emulation driver"
+	default y
+	depends on DT_HAS_ZEPHYR_OTP_EMUL_ENABLED
+	help
+	  Enable OTP emulator driver used for testing purposes.
```

Comment:
maybe something like this:
```Suggestion
	  Enable OTP emulator driver used for testing purposes.
	  It will use an static RAM buffer per instance which size is defined
	  in the DT node.
```